guacamole-common-js/src/main/webapp/modules/InputStream.js

```diff
@@ -76,4 +76,66 @@ Guacamole.InputStream = function(client, index) {
         client.sendAck(guac_stream.index, message, code);
     };
 
+    /**
+     * Creates a new ReadableStream that receives the data sent to this stream
+     * by the Guacamole server. This function may be invoked at most once per
+     * stream, and invoking this function will overwrite any installed event
+     * handlers on this stream.
+     * 
+     * A ReadableStream is a JavaScript object defined the "Streams" standard.
```

Comment:
Missing either `in` or `by` in this comment: `defined in the "Streams" standard.`